src/biogeophys/CanopyHydrologyMod.F90

```diff
@@ -645,12 +674,12 @@ subroutine FracWet(numf, filter, canopystate_inst, waterstatebulk_inst, waterdia
 
              if (h2ocan > 0._r8) then
                 vegt    = frac_veg_nosno(p)*(elai(p) + esai(p))
-                dewmxi  = 1.0_r8/dewmx(p)
+                dewmxi  = 1.0_r8/params_inst%dewmx  ! wasteful division
                 fwet(p) = ((dewmxi/vegt)*h2ocan)**0.666666666666_r8
                 fwet(p) = min (fwet(p),maximum_leaf_wetted_fraction)   ! Check for maximum limit of fwet
                 if (snocan(p) > 0._r8) then
-                   dewmxi  = 1.0_r8/dewmx(p)
-                   fcansno(p) = ((dewmxi/(vegt*6.0_r8*10.0_r8))*snocan(p))**0.15_r8 ! must match snocanmx 
+                   dewmxi  = 1.0_r8/params_inst%dewmx  ! wasteful division
+                   fcansno(p) = ((dewmxi / (vegt * params_inst%sno_stor_max * 10.0_r8)) * snocan(p))**0.15_r8 ! must match snocanmx 
```

Comment:
@swensosc @olyson @dlawrenncar , this is the second line that I referred to in my comment to you a few seconds ago.